stable/signalfx-agent/values.yaml

```diff
@@ -0,0 +1,196 @@
+# Version of the signalfx-agent to deploy.  This will be the default for the
+# docker image tag if not overridden with imageTag
+agentVersion: 3.0.0
+
+# The access token for SignalFx.  REQUIRED
+signalFxAccessToken:
+
+# Image pull policy for the agent pod
+pullPolicy: IfNotPresent
+
+# The docker image to use for the agent
+imageRepository: quay.io/signalfx/signalfx-agent
```

Comment:
Check our review guidelines for how to configure the image.
https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md